### PR TITLE
Improvements to prevent resources bug

### PR DESCRIPTION
**Pull request by xavierraffin**

### CODE REVIEW

**Comment by buildbotctp** [comment]
Can one of the admins verify this patch?

**Comment by antoine-de** [comment]
Jenkins, test this please

File: source/jormungandr/jormungandr/interfaces/v1/ResourceUri.py [hunk 1]
```diff
@@ -44,14 +44,16 @@
 
 class ResourceUri(StatedResource):
 
-    def __init__(self, authentication=True, *args, **kwargs):
+    def __init__(self, authentication=True, links=True, *args, **kwargs):
         StatedResource.__init__(self, *args, **kwargs)
         self.region = None
-        self.method_decorators.append(add_id_links())
+        if (links):
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/ResourceUri.py]: in python you don't need the `()` in the if

**Comment by antoine-de** [comment]
apart from my small lint remark, it's OK for me, thanks @xavierraffin !

**Pull request merged**